react/finger-exercises/ecmascript/02 - promises/index.js

```diff
@@ -0,0 +1,17 @@
+// Hint: use setInterval, create a new Promise and measure time with Date.now()
+
+export function delay(delayTime) {
+  return new Promise(((resolve) => {
+    setTimeout(() => {
```

Comment:
the second argument for `setTimeout` is the delay time in milliseconds. You can find more information here: https://www.w3schools.com/jsref/met_win_settimeout.asp
If you test this using `console.time()` and `console.timeEnd()` the promise is not being resolved after the `delayTime` ms. Try to change this so that the `resolve` function is executed after `delayTime`  ms